src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -157,6 +159,21 @@ This resets the cluster to its initial state.`,
 		Short: "Forward a port on the local machine to pachd. This command blocks.",
 		Long:  "Forward a port on the local machine to pachd. This command blocks.",
 		Run: cmdutil.RunFixedArgs(0, func(args []string) error {
+			pidfile.SetPidfilePath("~/.pachyderm.port.forward.pid")
+			pid, err := pidfile.Read()
+			if err != nil && !os.IsNotExist(err) {
+				return err
+			}
+			if pid != 0 {
+				if err := syscall.Kill(-pid, syscall.SIGKILL); err != nil {
+					if !strings.Contains(err.Error(), "no such process") {
+						return err
```

Comment:
I think in this case we should print a warning, but don't want to err, right? Otherwise, you'll never be able to run port forward bcz the process that it's trying to kill has already exited.